plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -28,8 +28,10 @@ import kotlin.reflect.KClass
 
 const val VERIFICATION_GROUP = "Verification"
 const val FORMATTING_GROUP = "Formatting"
+const val HELP_GROUP = "Help"
```

Comment:
I think a few of these already exist as constants in the Gradle public API. Any thoughts on simply using them instead?

Reply:
Wasn't aware of them, can you point where I can find them?

Reply:
So far only found in `HelpTasksPlugin.HELP_GROUP`

Reply:
I've add using `HelpTaskPlugin.HELP_GROUP` constant.

Reply:
`LifecycleBasePlugin.VERIFICATION_GROUP`

Reply:
added